hangman.py

```diff
@@ -0,0 +1,250 @@
+# Problem Set 2, hangman.py
+# Name: Zlotenko Bohdana
+# Students group: KM-04
+# Collaborators: Alexey Garmash
+# Time spent: really a lot
+
+import random
+from string import ascii_lowercase
+
+WORDLIST_FILENAME = "words.txt"
+VOWELS = ('a', 'e', 'i', 'o', 'u')
+UNKNOWN_LETTER = "_ "
+HINT = "*"
+
+warnings = 3
+guesses = 6
```

Comment:
BTW - it would be good to have a constant for these initial values, like `WARNINGS_INITIAL`